src/lib/serviceRoutines/postIndividualContextEntityAttribute.cpp

```diff
@@ -61,7 +61,8 @@ std::string postIndividualContextEntityAttribute
                                                                 &parseDataP->upcar.res,
```

Comment:
CHANGE_NEXT_RELEASE entry missing.

Maybe should add some test harness to check this?